vane-proxy-core/src/main/java/org/oddlama/vane/proxycore/ManagedServer.java

```diff
@@ -0,0 +1,174 @@
+package org.oddlama.vane.proxycore;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.Random;
+
+@SuppressWarnings("unused")
+public class ManagedServer {
+
+	private String id;
+	String display_name;
+	String favicon;
+	Quotes quotes;
+	Motd motd;
+	ServerStart start;
+
+	public ManagedServer() {}
+
+	public String getDisplay_name() {
+		return display_name;
+	}
+
+	public void setDisplay_name(String display_name) {
+		this.display_name = display_name;
+	}
+
+	public String getFavicon() {
+		return favicon;
+	}
+
+	public void setFavicon(String favicon) {
+		this.favicon = favicon;
+	}
+
+	public Quotes getQuotes() {
+		return quotes;
+	}
+
+	public void setQuotes(Quotes quotes) {
+		this.quotes = quotes;
+	}
+
+	public Motd getMotd() {
+		return motd;
+	}
+
+	public void setMotd(Motd motd) {
+		this.motd = motd;
+	}
+
+	public ServerStart getStart() {
+		return start;
+	}
+
+	public void setStart(ServerStart start) {
+		this.start = start;
+	}
+
```

Comment:
Just a note, all these getters/setters are needed by SnakeYAML. I haven't found a way to avoid them, they are pretty hideous though.

Reply:
Thankfully I figured it out :smile:.